<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserMyBooks</class>
 <widget class="QMainWindow" name="UserMyBooks">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Мои книги</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>291</width>
      <height>251</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Plain</enum>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="6" column="1">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>30</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="1" colspan="3">
      <widget class="QLineEdit" name="leISBN">
       <property name="maxLength">
        <number>13</number>
       </property>
       <property name="placeholderText">
        <string>Введите ISBN</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0" colspan="2">
      <widget class="QPushButton" name="btnSave">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Сохранить</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lbLit">
       <property name="text">
        <string>Произведение</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="lbDescription">
       <property name="text">
        <string>Описание</string>
       </property>
      </widget>
     </item>
     <item row="8" column="2" colspan="2">
      <widget class="QPushButton" name="btnDel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Удалить</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="3">
      <widget class="QComboBox" name="cbLit">
       <property name="placeholderText">
        <string>Произведение</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lbISBN">
       <property name="text">
        <string>ISBN</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="3">
      <widget class="QTextEdit" name="teDescription">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>100</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="4">
      <widget class="QCheckBox" name="chbIsOffered">
       <property name="text">
        <string>Объявление открыто</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="btnMenu">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>540</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Меню</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>681</width>
      <height>291</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="vlTable">
     <item>
      <layout class="QHBoxLayout" name="hlSearch">
       <item>
        <widget class="QCheckBox" name="chbIsOfferedSearch">
         <property name="text">
          <string>Объявление открыто</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leSearch">
         <property name="placeholderText">
          <string>Введите название</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnSearch">
         <property name="text">
          <string>Поиск</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableWidget" name="twData">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="btnAddMode">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>291</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить книгу</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="twReviews">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>350</y>
      <width>881</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbReviewsTitle">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>310</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Отзывы</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
